contracts/AMMWrapper.sol

```diff
@@ -231,7 +214,11 @@ contract AMMWrapper is IAMMWrapper, StrategyBase, ReentrancyGuard, BaseLibEIP712
      * @dev internal function of `trade`.
      * It executes the swap on chosen AMM.
      */
-    function _prepare(AMMLibEIP712.Order memory _order, InternalTxData memory _internalTxData) internal {
+    function _prepare(
```

Comment:
nit: Maybe could be renamed to something like `_transferTakerAssetToAMM` to reveal its intention.